src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -219,6 +219,14 @@ def get_env(self):
         ofi_interface = None
         ofi_domain = None
         ofi_share_addr = None
+        hg_log_subsys = None
+        hg_log_level = None
+
+        if "HG_LOG_SUBSYS" in os.environ:
+            hg_log_subsys = os.environ.get("HG_LOG_SUBSYS")
+
+        if "HG_LOG_LEVEL" in os.environ:
+            hg_log_level = os.environ.get("HG_LOG_LEVEL")
```

Comment:
FYI - the default is `None` so you could just do 
```suggestion
        hg_log_subsys = os.environ.get("HG_LOG_SUBSYS")
        hg_log_level = os.environ.get("HG_LOG_LEVEL")
```

Reply:
> FYI - the default is `None` so you could just do

Sorry, no was not meant for review, forgot to make it a draft, and was a quick copy/paste of existing envars how they were set.  It's good to know though, i ll make changes in a separate PR where I want to export those 2 additional envars in the test

Reply:
these changes moved to https://github.com/daos-stack/daos/pull/12161